code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -316,6 +314,13 @@
 			O.shield_key = TRUE
 			depotarea.shields_up()
 
+/mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/armory/death() // i hope it's all right
+	if(!istype(depotarea))
+		return ..()
+	if(!length(depotarea.shield_list)) // not opening lockers without getting shields down
+		depotarea.unlock_lockers()
```

Comment:
If I understand this logic correctly, this means that if you somehow kill the QM before killing the officer to lower shields, the lockers will be inaccessible short of breaking them. Is this intended?

Reply:
Yes, it is intented. If you managed to get to the armory without killing an officer (like being a cultist with a shifter, vampire with a blood pool etc), you'll get a raise_alert() for killing QM and will have to hack/destroy the closets to get the loot, so it's kinda like wasting an extra minute on it

Reply:
Correct me if I'm wrong but doesn't the depot just immediately delta on using any form of teleport?

Reply:
> doesn't the depot just immediately delta on using any form of teleport?

no, you can use veil shifter, cult's teleport spell, any magic from wiz, vampire teleports etc. The only things that are forbidden (you can't use them at all) are as simple as hand teles, jaunters and any other cheap non-magical stuff. No `raise_alert()` on using teleports